backend/src/main/java/com/soen341/instagram/model/Account.java

```diff
@@ -15,7 +16,7 @@
     @NotNull
     private String email;
 
-    @NotNull
+    @NotBlank(message="Password cannot be blank")
```

Comment:
What's the use of having a message defined in `@NotBlank()` if it's the same as the one in `PasswordBlankException`

Reply:
For formality, this just ensure that if for some reason you implement a function that try to save to database and you forget to handle the exception. This would catch it and prevent issue in db.

Reply:
Another reason why I handled it with `PasswordBlankException` and not simply relies on the annotation `@NotBlank` is because we use `PasswordEncoder` so when you pass an empty string it's encode so you'll get a random String e.g `asdksadsakdjaksjdje213213k`, this encoded string isn't blank anymore. That was our initial bug.